package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "test": "nodeunit test.js"
+    "test": "./node_modules/.bin/nodeunit test.js"
```

Comment:
But I had a global nodeunit. Doing it this way seems better in some cases and worse in none.